.github/workflows/build.yml

```diff
@@ -143,7 +153,17 @@ jobs:
         shell: bash
         run: |
           cd litmus-portal/upgrade-agents/control-plane
-          docker build . -f Dockerfile --build-arg TARGETARCH=amd64
+          docker build . -f Dockerfile -t docker.io/litmuschaos/upgrade-agent-cp:${{ github.sha }} --build-arg TARGETARCH=amd64
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'docker.io/litmuschaos/upgrade-agent-cp:${{ github.sha }}'
```

Comment:
Trivy is checking the images before the push. Can we check the local build images?

Reply:
We are building the images locally and checking the build in trivy.

Reply:
But it's not being pushed right? This will pull "docker.io/litmuschaos/upgrade-agent-cp:${{ github.sha }}" directly from docker hub

Reply:
The purpose is to test the image vulnerability for every commit. 

We have build docker.io/litmuschaos/upgrade-agent-cp:${{ github.sha }} then running trivy scan on it we are not pulling it from anywhere.